components/cloud-event-gateway-proxy/cmd/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/kelseyhightower/envconfig"
+	"go.uber.org/zap"
+
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/gateway"
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/handler"
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/oauth"
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/receiver"
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/sender"
+	"github.com/kyma-project/kyma/components/cloud-event-gateway-proxy/pkg/signals"
+)
+
+const (
+	// Constants for the underlying HTTP Client transport. These would enable better connection reuse.
+	defaultMaxIdleConns        = 1000
+	defaultMaxIdleConnsPerHost = 1000
+
+	// emsPublishEndpoint the endpoint used to publish cloudevents
+	emsPublishEndpoint = "/events"
+)
+
+func main() {
+	logger, err := zap.NewProduction()
```

Comment:
any reason to use zap? zap vs logrus? logrus json formatter gives us a nice way to parse the logs for debugging.

Reply:
Zap can do this as well ;)

```go
	log, err := zap.NewDevelopment()
	if err != nil {
		panic(err)
	}
```

Output:
```
2020-09-02T10:35:58.008+0200    INFO    http-adapter/main.go:28 foo     {"key": "value"}
```

Reply:
Changed to `logrus`.